src/helpers/discord_utils.ts

```diff
@@ -898,6 +944,13 @@ export async function generateOptionsMessage(
         .filter((option) => optionStrings[option as GameOption])
         .filter((option) => !PriorityGameOption.includes(option as GameOption));
 
+    // Move answer out from the priority options
+    if (isSpotify) {
```

Comment:
What is this doing?

Reply:
This clears the first options field and moves the ,spotify/,answer fields to the start. I'll update the comment.